[2025-02-16T21:57:51.858+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: kafka_spark_dag.kafka_data_stream manual__2025-02-16T21:57:43.190949+00:00 [queued]>
[2025-02-16T21:57:51.945+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: kafka_spark_dag.kafka_data_stream manual__2025-02-16T21:57:43.190949+00:00 [queued]>
[2025-02-16T21:57:51.946+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2025-02-16T21:57:52.022+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): kafka_data_stream> on 2025-02-16 21:57:43.190949+00:00
[2025-02-16T21:57:52.055+0000] {standard_task_runner.py:57} INFO - Started process 4687 to run task
[2025-02-16T21:57:52.104+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'kafka_spark_dag', 'kafka_data_stream', 'manual__2025-02-16T21:57:43.190949+00:00', '--job-id', '27', '--raw', '--subdir', 'DAGS_FOLDER/dag_kafka_spark.py', '--cfg-path', '/tmp/tmp60hez2kj']
[2025-02-16T21:57:52.110+0000] {standard_task_runner.py:85} INFO - Job 27: Subtask kafka_data_stream
[2025-02-16T21:57:52.529+0000] {task_command.py:416} INFO - Running <TaskInstance: kafka_spark_dag.kafka_data_stream manual__2025-02-16T21:57:43.190949+00:00 [running]> on host de5e2408bcb1
[2025-02-16T21:57:53.151+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='kafka_spark_dag' AIRFLOW_CTX_TASK_ID='kafka_data_stream' AIRFLOW_CTX_EXECUTION_DATE='2025-02-16T21:57:43.190949+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-16T21:57:43.190949+00:00'
[2025-02-16T21:58:07.647+0000] {logging_mixin.py:154} INFO - Dataset downloaded successfully.
[2025-02-16T21:58:07.790+0000] {logging_mixin.py:154} INFO - Dataset extracted successfully.
[2025-02-16T21:58:07.792+0000] {logging_mixin.py:154} INFO - No CSV file found in extracted data.
[2025-02-16T21:58:07.795+0000] {logging_mixin.py:154} INFO - Dataset preparation failed.
[2025-02-16T21:58:07.796+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-02-16T21:58:07.855+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=kafka_spark_dag, task_id=kafka_data_stream, execution_date=20250216T215743, start_date=20250216T215751, end_date=20250216T215807
[2025-02-16T21:58:07.915+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-02-16T21:58:07.996+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
